external/radiation/tests/test_driver.py

```diff
@@ -193,9 +201,15 @@ def test_radiation_valiation():
         randomdict = io.load_random_numbers(LOOKUP_DIR, rank)
         lwdict = io.load_lw(LOOKUP_DIR)
         swdict = io.load_sw(LOOKUP_DIR)
+        solcon = Model.pop("solcon")
+        solhr = Model.pop("solhr")
+
+        gfs_physics_control = get_gfs_physics_control(Model)
```

Comment:
It'd be nice to abandon this test that uses serialized data instead of having to keep modifying it to work with the port as we change it, but at the moment we don't have another validation test of the port. At some point when it validates "perfectly" or close to it in the prognostic run we can use that, but we're not there quite yet.